cas-server-core-tickets/src/main/java/org/apereo/cas/ticket/registry/AbstractTicketRegistry.java

```diff
@@ -46,37 +27,18 @@
  * This is a published and supported CAS Server API.
  * </p>
  */
-public abstract class AbstractTicketRegistry implements TicketRegistry, TicketRegistryState, Job {
+public abstract class AbstractTicketRegistry implements TicketRegistry, TicketRegistryState {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This class has too many methods, consider refactoring it.](https://www.codacy.com/app/mmoayyed/cas/file/2749420056/issues/source?bid=3256115&fileBranchId=3357337#l30)